tasks/service-runner/config/asf-service-sidecar.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
```

Comment:
Would love to see this templatized or something rather than a 106 line YAML for every version of every service.  File a maintenance issue?

Reply:
yeah - we haven't filed an issue yet, but Chris and I have both mentioned that we need to templatize away all the redundant boilerplate

Reply:
Filed ticket https://bugs.earthdata.nasa.gov/browse/HARMONY-935 to do this.